CHANGES.txt

```diff
@@ -39,93 +39,93 @@ Breaking Changes
 
-- Added support to manually control the allocation of shards using
-  ``ALTER TABLE REROUTE``.
-  Supported reroute-options: MOVE, ALLOCATE REPLICA, CANCEL
+ - Added support to manually control the allocation of shards using
+   ``ALTER TABLE REROUTE``.
+   Supported reroute-options: MOVE, ALLOCATE REPLICA, CANCEL
 
-- Added new system table ``sys.allocations`` which lists shards and their
-  allocation state including the reasoning why they are in a certain state.
+ - Added new system table ``sys.allocations`` which lists shards and their
+   allocation state including the reasoning why they are in a certain state.
 
-- Upgraded Elasticsearch to v5.6.3
+ - Upgraded Elasticsearch to v5.6.3
 
-- Added new table setting ``allocation.max_retries`` that defines the number of
-  attempts to allocate a shard before giving up and leaving it unallocated.
+ - Added new table setting ``allocation.max_retries`` that defines the number of
+   attempts to allocate a shard before giving up and leaving it unallocated.
 
-- Function arguments are now linked to each other, where possible. This enables
-  type inference between arguments such that arguments can be converted to match
-  a function's signature. For example, ``coalesce(integer, long)`` would have
-  resulted in an "unknown function" message. We now convert this call into
-  ``coalesce(long, long)``. The conversion is possible through a type precedence
-  list and convertibility checks on the data types.
+ - Function arguments are now linked to each other, where possible. This enables
+   type inference between arguments such that arguments can be converted to match
+   a function's signature. For example, ``coalesce(integer, long)`` would have
+   resulted in an "unknown function" message. We now convert this call into
+   ``coalesce(long, long)``. The conversion is possible through a type precedence
+   list and convertibility checks on the data types.
 
-- Functions which accept regular expression flags now throw an error when
-  invalid flags are provided.
+ - Functions which accept regular expression flags now throw an error when
+   invalid flags are provided.
 
-- Added the ``pg_backend_pid()`` function for enhanced PostgreSQL compatibility.
+ - Added the ``pg_backend_pid()`` function for enhanced PostgreSQL compatibility.
 
-- Added support for the PSQL ParameterDescription message which allows to get
-  the parameter types in prepared statements up front without specifying the
-  actual arguments first. This fixes compatibility issues with some drivers.
-  This works for the most common use cases except for DDL statements.
+ - Added support for the PSQL ParameterDescription message which allows to get
+   the parameter types in prepared statements up front without specifying the
+   actual arguments first. This fixes compatibility issues with some drivers.
+   This works for the most common use cases except for DDL statements.
 
-Fixes
-=====
+ Fixes
+ =====
```

Comment:
this is wrong

Reply:
@nslater thanks for the catch. Fixup pushed.